docs/users/submit-eruption.md

```diff
@@ -17,15 +17,15 @@ To enter an observed eruption, you need the following information:
 * the time accuracy of your observation, i.e., 
     * exact, if you observed the eruption from the start, 
     * near start, if you observed the eruption within a minute of the start, 
-    * in eruption, if you oberved the eruption, but not from the start, 
-    * electronic, if the eruption was determined by electronic equipement (e.g. temperature logger or seismometer), 
-    * approximate, if you observed the eruption, but did not note the exact time and therefore approximate it, or if you know an eruption has occured, but do not know its start time (e.g. for overnight eruptions), 
+    * in eruption, if you observed the eruption, but not from the start, 
+    * electronic, if the eruption was determined from monitoring data (e.g. temperature logger or seismometer), 
+    * approximate, if you observed the eruption, but did not note the exact time and therefore approximate it, or if you know an eruption has occurred, but do not know its start time (e.g. for overnight eruptions), 
 * (optionally) the duration of the eruption, 
 * as required additional labels, i.e., 
-    * the webcam label, if you observed the eruption on the webcam, 
-    * the intial label, if you observed the initial eruption of a series of a geyser known to erupt in series (e.g. Lion),
-    * the questionable label, if you think an eruption occured, but aren't sure, 
-    * the major or minor label (or the long or short label for Old Faithful), if you observed a major or minor eruption of a geyser known to erupt in a bimodal way (e.g. Old Faithful or Castle),
+    * the webcam label, if you observed the eruption on one of the [Old Faithful webcams](https://www.nps.gov/yell/learn/photosmultimedia/webcams.htm)., 
```

Comment:
Same as in glossary: I would keep this more general as other webcams have existed and hopefully will exist in the future (still hoping for a webcam to view Pohutu on 😉).